drivers/SmartThings/zigbee-motion-sensor/src/aqara/aqara_utils.lua

```diff
@@ -20,28 +20,23 @@ local PREF_CHANGED_VALUE = "prefChangedValue"
 local PREF_FREQUENCY_KEY = "prefFrequency"
 local PREF_FREQUENCY_VALUE_DEFAULT = 60
 
-local function custom_attribute(device, cluster_id, attribute_id)
-  local message = cluster_base.read_attribute(device, data_types.ClusterId(cluster_id), attribute_id)
+local function read_custom_attribute(device, attribute_id)
+  local message = cluster_base.read_attribute(device, data_types.ClusterId(PRIVATE_CLUSTER_ID), attribute_id)
```

Comment:
you can just use `cluster_base.read_manufacturer_specific_attribute`

Reply:
I updated.